lib/spack/docs/tutorial_buildsystems.rst

```diff
@@ -103,7 +103,7 @@ This will open the :code:`AutotoolsPackage` file in your text editor.
     long examples. We only show what is relevant to the packager.
 
 
-.. literalinclude:: tutorial/examples/Autotools/autotools_class.py
```

Comment:
This could have been done before, and being done now to avoid updating multiple files for the same change. Similarly in the equivalent changes for other classes.